pegasus/sites.v3/code.org/public/mc.haml

```diff
@@ -50,6 +50,10 @@ social:
   =view :inspirational_videos
 
 %h2.mc-header2
+  = I18n.t(:are_you_teacher)
+  %a{href: "/hourofcode/mc"}= I18n.t(:review_lesson_plan)
```

Comment:
Discussed offline, but for onlookers this is the same as the "Teacher's Notes" link on /learn. Typically it's only to a lesson plan, but for our stuff we have additional resources so it goes to this page that has both the lesson plan and extra stuff.